odoo/addons/base/models/ir_property.py

```diff
@@ -110,12 +110,11 @@ def write(self, values):
         # if any of the records we're writing on has a res_id=False *or*
         # we're writing a res_id=False on any record
         default_set = False
+        values = self._update_values(values)
```

Comment:
Is it useful ? `_update_values` doesn't change `res_id`. Also, it looks like you call `_update_values` twice now.

Reply:
Indeed the `_update_values` can be done only once before and not in the write